traptor/traptor.py

```diff
@@ -216,10 +226,16 @@ def _create_birdy_stream(self):
                 self.birdy_stream = self.birdy_conn.stream.statuses.filter.post(locations=self.twitter_rules,
                                                                                 stall_warnings='true')
             except TwitterApiError as e:
-                self.logger.critical(e)
+                self.logger.critical("Caught Twitter Api Error", extra={
+                    'error_type': 'TwitterAPIError',
+                    'ex': traceback.format_exc()
+                })
                 sys.exit(3)
         else:
-            self.logger.critical('That traptor type has not been implemented')
+            self.logger.critical('Caught error creating birdy stream for Traptor type that does not exist', extra ={
+                'error_type': 'NotImplementedError',
+                'ex': traceback.format_exc()
```

Comment:
This `else` block does not result in a traceback since its just the last block of an `if/elif/else` conditional.  I'm not sure what happens with `ex:  traceback.format_exc()` call.  My guess is that this field will always be `'None\n`.